src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under construction';
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state,
```

Comment:
- [ ] The checkStatus action should receive a payload which checks if the categories is under construction, if not it should return the categories state. see the code snippet below as a guide 
- code snippet
```javascript
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    }
```

Reply:
Dear @Efiamotu-1 👋🏾
Thanks for the suggestion, and feedback,
I appreciate it,
I already modified this part.
Best regards,